husky_components/navigation/main.js

```diff
@@ -253,7 +249,11 @@ define(function() {
             // load Data
             if (!!this.options.url) {
                 this.sandbox.util.load(this.options.url, null, 'json')
-                    .then(this.render.bind(this))
+                    .then(function(data) {
+                        this.options.data = data;
```

Comment:
Yes, it was there before. The navigation componenent is something which we really should consider refactoring. However, it was just to much for this task and PR to refactor bad things.